README.adoc

```diff
@@ -108,15 +105,14 @@ mvn liberty:stop-server
 Now, navigate back to the `start` directory to begin.
 
 // =================================================================================================
-// Adding Microprofile Metrics to the inventory microservice
+// Adding MicroProfile Metrics to the inventory service
 // =================================================================================================
 
-== Adding Microprofile Metrics to the inventory microservice
+== Adding MicroProfile Metrics to the inventory service
 
 The MicroProfile Metrics API was added as a dependency to your `pom.xml` file. Look for the dependency with
-the `org.eclipse.microprofile.metric` group ID. This feature allows you to use the MicroProfile Metrics API to provide
-metrics from your microservices. Also, in the `pom.xml` file, you will find the `javax.net.ssl.trustStore` system
-property variable. This variable points to the SSL certificate of the system.
+the `org.eclipse.microprofile.metric` group ID. This dependency enables you to use MicroProfile Metrics
```

Comment:
What about..."Use this dependency to include the MicroProfile Metrics API in your code..."